### PR TITLE
Add Via support to Plexus75 PCB

### PR DESCRIPTION
**Pull request by npspears**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/plexus75/config.h [hunk 1]
```diff
@@ -23,7 +23,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define PRODUCT_ID      0x5338
 #define DEVICE_VER      0x0001
 #define MANUFACTURER    nasp
-#define PRODUCT         Plexus75
+#define PRODUCT         nasp Plexus75
```
Comment (drashna) [inline keyboards/plexus75/config.h:26]: https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs
> don’t repeat MANUFACTURER in the PRODUCT value


```suggestion
#define PRODUCT         Plexus75
```

File: keyboards/plexus75/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+     */
+    [_FN1] = LAYOUT_2x2u(
+        _______, _______, RGB_TOG, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, _______, _______, _______, _______, _______, _______, _______, KC_PSCR,
+        KC_GRV,     KC_1,   KC_2,     KC_3,    KC_4,    KC_5, _______, _______, _______,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0, _______,
+        KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, _______, _______, _______, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_BSPC,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MUTE, KC_VOLD, KC_VOLU, KC_BSLS, _______, _______,
+        RESET, RESET, _______, _______, _______, _______, _______, _______, _______, _______, _______
+    ),
+
+};
```
Comment (drashna) [inline keyboards/plexus75/keymaps/via/keymap.c:94]: Unless you've set the number of layers to be less than 4, you need to have 4 layers defined in the keymap, or the layer will be filled with gibberish. 

```suggestion
    [3] = LAYOUT_2x2u(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
    ),
};
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/plexus75/config.h (hunk 1 above)
Comment (npspears) [reply keyboards/plexus75/config.h]: ok

File: keyboards/plexus75/keymaps/via/keymap.c (hunk 2 above)
Comment (npspears) [reply keyboards/plexus75/keymaps/via/keymap.c:94]: will do

**Review by npspears — COMMENTED** [review]
edited files per drashna's corrections

File: keyboards/plexus75/rules.mk [hunk 3]
```diff
@@ -20,3 +20,4 @@ RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
 BLUETOOTH_ENABLE = no       # Enable Bluetooth
 AUDIO_ENABLE = no           # Audio output
 UNICODE_ENABLE = yes        # Unicode
+VIA_ENABLE = yes	        # Via
```
Comment (fauxpark) [inline keyboards/plexus75/rules.mk:23]: This needs to go at keymap level, not keyboard level (ie. `keymaps/via/rules.mk`).

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by npspears — COMMENTED** [review]
changing location of rules.mk now

File: keyboards/plexus75/keymaps/via/rules.mk [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+UNICODE_ENABLE = yes        # Unicode
```
Comment (fauxpark) [inline keyboards/plexus75/keymaps/via/rules.mk:22]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by npspears — COMMENTED** [review]
rules.mk change made

File: keyboards/plexus75/keymaps/via/rules.mk [hunk 5]
```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes            # Enable yes
```
Comment (fauxpark) [inline keyboards/plexus75/keymaps/via/rules.mk:1]:
```suggestion
VIA_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by npspears — COMMENTED** [review]
all changes made

File: keyboards/plexus75/keymaps/via/keymap.c (hunk 2 above)
Comment (fauxpark) [reply keyboards/plexus75/keymaps/via/keymap.c:94]: This is still outstanding. Otherwise LGTM.

Comment (npspears) [reply keyboards/plexus75/keymaps/via/keymap.c:94]: Sorry. Github noob. Think I've fixed it now.

**Review by npspears — COMMENTED** [review]
Final change made

**Review by fauxpark — APPROVED** [review]

**Review by ridingqwerty — APPROVED** [review]
Looks good, thanks.

**Pull request merged**